packages/sveltekit/src/vite/detectAdapter.ts

```diff
@@ -0,0 +1,65 @@
+import type { Package } from '@sentry/types';
+import * as fs from 'fs';
+import * as path from 'path';
+
+/**
+ * Supported @sveltejs/adapters-[adapter] SvelteKit adapters
+ */
+export type SupportedSvelteKitAdapters = 'node' | 'auto' | 'vercel' | 'other';
```

Comment:
[adapter-auto](https://kit.svelte.dev/docs/adapter-auto) and adapter-vercel can both be used to deploy to Vercel. Hence we permit both of them.